README.md

```diff
@@ -42,9 +40,6 @@ Notes:
 
 - Workspaces are defined in the `package.json` at the root
 - While workspaces allow the sequential running of commands across all workspaces, console is using a project `workspaces-run` to run command across all workspaces concurrently.
-- Installing a new dependency for a workspace must be done at the root using the `--workspace` parameter.
-  - `npm install some-lib --workspace frontend`
-  - This is because npm hoists up the dependencies and updates the root package-lock.json.
```

Comment:
Is there no similar restriction with yarn?

Reply:
i think yarn takes care of it for you